components/brave_rewards/browser/rewards_service_observer.h

```diff
@@ -51,6 +51,8 @@ class RewardsServiceObserver : public base::CheckedObserver {
                              brave_rewards::ContentSiteList) {};
   virtual void OnPublisherBanner(brave_rewards::RewardsService* rewards_service,
                                  const brave_rewards::PublisherBanner banner) {};
+  virtual void OnRewardsMainEnabled(brave_rewards::RewardsService* rewards_service,
```

Comment:
we shouldn't be adding any method to the observer. Please use a callback instead